R/module_teal.R

```diff
@@ -113,6 +113,12 @@ ui_teal <- function(id,
       tags$div(
         footer,
         teal.widgets::verbatim_popup_ui(ns("sessionInfo"), "Session Info", type = "link"),
+        br(),
+        if (!inherits(session, "MockShinySession")) {
+          teal.widgets::verbatim_popup_ui(ns("lockFile"), "Compute .lock file", type = "link")
```

Comment:
I can see that you are using `verbatim_popup_ui` that has preview + copy + download. Are we really interested in all of these? I was thinking more about just enabling the download action. IMO the file contains a large number of lines and is not necessarily intended for human reading

Reply:
Would you trust an app that downloads some file on your system @pawelru ?
Presenting the content and the ability to copy the content on your own looks safer. This was my reasoning :)

Reply:
if this is requested - I would say yes
we already do this for reporter when user expclisitly click download
here it could be the same -> "Download .lock file"

Reply:
we have a download link now 90fbadbced4bc79959caba85684a33c0fb9382cd